plugins/generator-1.20.6/neoforge-1.20.6/templates/elementinits/potioneffects.java.ftl

```diff
@@ -65,9 +64,8 @@ package ${package}.init;
 
 	private static void expireEffects(Entity entity, MobEffectInstance effectInstance) {
 		<#compress>
-		MobEffect effect = effectInstance.getEffect();
 		<#list effects_that_expire as effect>
-		if (effect == ${effect.getModElement().getRegistryNameUpper()}.get()) {
+		if (!effectInstance.getEffect().isEmpty() && effectInstance.getEffect().is(${effect.getModElement().getRegistryNameUpper()}.get())) {

```

Comment:
This is not ok as you are comparing Holder with its value.  I can fix this when I have time if you feel you won't figure it out :)

Reply:
Ok thank you